### ISSUE TITLE
Jstyle file with special characters ReferenceHeaderParagraphFormat not working

### ISSUE THREAD

**Issue by teertinker** [JabRef/jabref#927]
- JabRef version (available in the About box): 3.3dev
- Operating system and version: Ubuntu 14.04 64 bit (the same behavior applies to Windows 10 )
- Steps to reproduce:
  1. Use LO build-in plugin
  2. choose a jstyle file, that uses special characters in ReferenceHeaderParagraphFormat and probably in ReferenceParagraphFormat
     like: ReferenceHeaderParagraphFormat="Überschrift0"
     [my jstyle file is encoded in UTF8]
  3. An error is raised:
     ![image](https://cloud.githubusercontent.com/assets/8830305/13666734/6572bdde-e6b3-11e5-8faf-557daab43914.png)

-> this was working properly in previous versions. If the style file needs a certain Encoding this should be documented.

**Comment by oscargus** [comment]
It has not been actively changed at least.

**Comment by Siedlerchr** [comment]
I could reproduce that, only if the jstyle file is in ansi format.
Converting it to UTF8  and it displays the name correctly

**Comment by teertinker** [comment]
Really strange... I can't recall that I changed the jstyle file. And it worked a year ago. I tried UTF8 and it keeps triggering the error. If I change encoding to "WINDOWS-1252" it works properly.

**Comment by Siedlerchr** [comment]
Hm. Try adding Überschrift0 again after converting it to UTF8. That seemed to help in my case.

**Comment by teertinker** [comment]
Not working for me. For further investigation: I uploaded my jstyle file (suffix changed to txt to enable uploading to github). It is saved in UTF8 and triggers the error on my machine.


[2campus_format.txt](https://github.com/JabRef/jabref/files/168715/2campus_format.txt)

**Comment by stefan-kolb** [comment]
What are the steps to reproduce?
If I select the style file in the OO dialog everything is working fine for me with your provided file.

**Comment by Siedlerchr** [comment]
You have to insert some cites in the OO doc and then click on synchronizing


2016-03-11 11:16 GMT+01:00 Stefan Kolb <notifications@github.com>:

**Comment by oscargus** [comment]
I think the main thing here is to have the SAME encoding in LO and JabRef,
not so important exactly which.

Not sure how to check/set encoding in LO though, but there's a clear risk
that it is harder to trigger the bug since new documents and style files
most likely use UTF-8 now.

**Comment by oscargus** [comment]
I tried to look into this and the only thing I found was: http://www.openoffice.org/api/docs/common/ref/com/sun/star/awt/CharSet.html which is deprecated.

In a major refactoring of the OO/LO support in #1047, all external style files are now loaded in the default encoding set in the Preference dialog (which makes sense since the Bib-file is most likely encoded like that, note that since several bib-files can be used they might have different encodings, so picking based on bib-file is hard...). Not sure if this helps.
